package.json

```diff
@@ -7,8 +7,8 @@
   ],
   "scripts": {
     "prepare": "husky install",
-    "test": "\"./node_modules/.bin/cucumber-js\" -p default",
-    "test:generators": "\"./node_modules/.bin/cucumber-js\" -p generators",
+    "test": "cross-env-shell ./node_modules/.bin/cucumber-js -p default",
```

Comment:
It's worth reviewing the docs for the [`npm run`](https://docs.npmjs.com/cli/v9/commands/npm-run-script#description) command -

> In addition to the shell's pre-existing PATH, npm run adds node_modules/.bin to the PATH provided to scripts. Any binaries provided by locally-installed dependencies can be used without the node_modules/.bin prefix. For example, if there is a devDependency on tap in your package, you should write:
 ```json
"scripts": {"test": "tap test/*.js"}
```
> instead of
```json 
"scripts": {"test": "node_modules/.bin/tap test/*.js"}
```

I believe that would remove the need for cross-env and fix this bug.

Reply:
This looks like exactly what we need. Thanks :) I've updated the PR(s).

Reply:
There can be a side note about the hardcoded usage of `node_modules/.bin` as well.


Someone might have a custom setup for the node_modules folder location. Currently, test entry points make the assumption of node_modules is in its default place. `npm bin` command provides the path of locally installed binaries dynamically.

Changing from

```json
"test": "node_modules/.bin/cucumber-js -p default"
```

to

```json
"test": "$(npm bin)/cucumber-js -p default"
```

addresses this and works fine on unix but I'm not sure about its cross-compatibility.